docs/assets/css/main.css

```diff
@@ -18602,3 +18608,70 @@ footer {
         visibility: hidden
     }
 }
+
+.footer-icons-list{
+    display:flex;
+    justify-content: center;
+}
+.footer-icons {
+    display: flex;
+    height: 0px;
+    justify-content: flex-start;
+    flex-direction: row;
+    padding: 4px;
+    position: relative;
+        
+}
+.footer-icons .footer-icons-text {
+    visibility: hidden;
+    width: 200px;
+  background-color:#3C494F;
+  color:white;
+  text-align: center;
+  border-radius: 6px;
+  padding: 5px;
+
+  position: absolute;
+  z-index: 1;
+  bottom: 45%;
+  left: 0px; 
+  margin-left: -30px;
+ 
+  
+}
+
+.footer-icons:hover .footer-icons-text {
+    visibility: visible;
+}
+
+  
+.social__link {
```

Comment:
It's ideal if we can reuse the same code from the layer5 site so as to reduce maintenance of two slightly different implementations. Are you able to use this sass instead? --> https://github.com/layer5io/layer5/blob/master/_sass/_hori-social.scss

Reply:
![Screenshot from 2019-12-21 20-44-52](https://user-images.githubusercontent.com/38222562/71309840-c3ca4800-2432-11ea-8031-b96f1898da63.png)
Are you sure you are updating the latest changes?

Reply:
and for sass part, i have done exactly same, i just changes the sass code into css

Reply:
Understood. Can we leave as sass and literally reuse the same code? This type of reuse practice improves sustainability and quality.

Reply:
okay will do.

Reply:
With respect to building the latest changes, yes, when I check out the branch (PR) and run `git log -1`, I see:

```
           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]
commit ace454a3d315f6786c040abea41d6d9111cb84b5 (HEAD -> pull/527)
Author: raghavaggarwal99 <ragha170103048@iitg.ac.in>
Date:   Sat Dec 21 17:06:59 2019 +0530

    Signedoff: raghavaggarwal99<ragha170103048@iitg.ac.in>

    Signed-off-by: raghavaggarwal99 <ragha170103048@iitg.ac.in>

    Signed-off-by: raghavaggarwal99 <ragha170103048@iitg.ac.in>
```
This is the same commit ID as is listed as the latest ID above. I see the same behavior in both Safari and Chrome.

Reply:
> okay will do.

thank you, sir. 👍